src/Main.java

```diff
@@ -39,10 +40,17 @@ public static void start(double dataGenStart, double dataGenEnd, double dataGenI
         System.out.println("Starting");
 
         ReadData data = new ReadData();
+        try{
+            ReadData.main(null);
+        }catch (IOException e){
+

```

Comment:
Do anything when we throw an error?  Print the stack trace at least? `e.printStackTrace();`